### ISSUE TITLE
TFLite v1.14.0 download_dependencies.sh is broken

### ISSUE THREAD

**Issue by hmeyer** [tensorflow/tensorflow#32054]
- OS Platform and Distribution (e.g., Linux Debian):
- TensorFlow installed from (source or binary): github
- TensorFlow version: v1.14.0
- Installed using virtualenv? pip? conda?: docker tensorflow/tensorflow:nightly-devel

```
$ git clone https://github.com/tensorflow/tensorflow
Cloning into 'tensorflow'...
remote: Enumerating objects: 38, done.
remote: Counting objects: 100% (38/38), done.
remote: Compressing objects: 100% (37/37), done.
remote: Total 668742 (delta 8), reused 29 (delta 1), pack-reused 668704
Receiving objects: 100% (668742/668742), 378.17 MiB | 28.86 MiB/s, done.
Resolving deltas: 100% (542405/542405), done.
Checking out files: 100% (20047/20047), done.
$ cd tensorflow/ 
$ git checkout v1.14.0
Checking out files: 100% (8943/8943), done.
Note: checking out 'v1.14.0'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b <new-branch-name>

HEAD is now at 87989f6959 Add Sergii Khomenko to contributor list
$ ./tensorflow/lite/tools/make/download_dependencies.sh
downloading http://mirror.tensorflow.org/bitbucket.org/eigen/eigen/get/a0d250e79c79.tar.gz

gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
```

**Comment by hmeyer** [comment]
Oh. Yes. That would be https://www.tensorflow.org/lite/guide/build_rpi

**Comment by hmeyer** [comment]
Sadly building from HEAD is also broken (at least for rpi armv6).
See https://github.com/tensorflow/tensorflow/issues/32055#issuecomment-525857017

**Comment by hmeyer** [comment]
Also default (armv7l) rpi build is broken at HEAD: https://github.com/tensorflow/tensorflow/issues/32055#issuecomment-525862626

**Comment by hubzhangchao** [comment]
edit download_dependencies.sh, then change 'http.*bitbucket.org' to 'https.*bitbucket.org', It worked;

**Comment by terryheo** [comment]
URLs are all updated. Let me close this.
Please, feel free to reopen if the problem still exists.

**Comment by terryheo** [comment]
FYI, updated URLs are available since TFLite v1.15.
